docs/source/validation.rst

```diff
@@ -232,6 +232,111 @@ like this::
         return 'Hello'
 
 
+Using WTForms
+~~~~~~~~~~~~~
+
+For using WTForms schema see :ref:`WTForms integration <wtforms>`.
+
+
+Validation of request.matchdict
+-------------------------------
+
+Sometimes it can be useful to validate types of variables passed through 
+**request.matchdict** e.g. when you have a Service method that depends upon 
+proper type of this variable.
+
+For example in CRUD-like services you often define methods (GET/PUT/DELETE) 
+whose URI includes an **id** that is used for querying the object. To ensure
+that your code doesn't break, you can validate coercion of the specified 
+**request.matchdict** variable to given type. For example::
+
+    from cornice import Service
```

Comment:
Ok. I'll move this code. Currently cornice/validators.py is almost empty, so I thought a better suited location would be schema.py.